dist/api_data.py

```diff
@@ -411,11 +411,11 @@ def __cmp__(self, other):
         enable insertion of code that interrupts the usual timing of
         operations with a goal of uncovering race conditions and unexpected
         blocking. This option is intended for use with internal stress
-        testing of WiredTiger. Only available if WiredTiger is configured
-        with --enable-diagnostic. Options are given as a list, such as
-        <code>"diagnostic_timing_stress=[checkpoint_slow]"</code>''',
+        testing of WiredTiger. Options are given as a list, such as
+        <code>"diagnostic_timing_stress=[checkpoint_slow,
```

Comment:
Please also change the name of the option to not reference diagnostic. `timing_stress_for_test` seems like a good choice to me.